src/test/kotlin/com/lapanthere/flink/api/kotlin/typeutils/SetTypeInformationTest.kt

```diff
@@ -0,0 +1,12 @@
+package com.lapanthere.flink.api.kotlin.typeutils
+
+import org.apache.flink.api.common.typeutils.TypeInformationTestBase
+
+class SetTypeInformationTest : TypeInformationTestBase<SetTypeInformation<*>>() {
+    override fun getTestData(): Array<SetTypeInformation<*>> = arrayOf(
+        createTypeInformation<Set<String>>() as SetTypeInformation<*>,
+        createTypeInformation<MutableSet<Pair<String, Int>>>() as SetTypeInformation<*>,
+        createTypeInformation<HashSet<DataClass>>() as SetTypeInformation<*>,
+        createTypeInformation<LinkedHashSet<Int>>() as SetTypeInformation<*>,
+    )
+}
```

Comment:
Could you add the missing new line and rebase so that CI will run for your PR?

Reply:
Fixed. Pushed changes

Reply:
Thank you!